lib/r2d2/util.rb

```diff
@@ -29,12 +29,13 @@ def generate_shared_secret(private_key, ephemeral_public_key)
       private_key.dh_compute_key(point)
     end
 
-    def derive_hkdf_keys(ephemeral_public_key, shared_secret, info)
+    def derive_hkdf_keys(ephemeral_public_key, shared_secret, info, key_length_bytes = 16)
```

Comment:
Should `key_length_bytes` have a default value?

Reply:
the function is also used by `android_pay_token` which I didn't want to touch, and 16 bytes was an original assumption of the function